break-monolith-apart2.adoc

```diff
@@ -796,13 +796,14 @@ Open the monolith UI by clicking the route URL icon (the arrow to the upper righ
 
 Observe that we are still using the old monolith, but leveraging the new catalog service:
 
-image::coolstore_web.png[Greeting, 700]
+image::coolstore_web_microservice.png[Greeting, 700]
```

Comment:
@danieloh30 it just so happens I do.  See [here](http://guides-m1-labs-infra.apps.cluster-v4lzs.v4lzs.sandbox565.opentlc.com/workshop/cloudnative/lab/break-monolith-apart2)

(The image is meant to be small and off to the side...it's trying to show something that shouldn't be there)

Reply:
It looks awkward to me since it's too small as a wrong resolution:
![image](https://user-images.githubusercontent.com/14066977/119291863-3ab5d280-bc1d-11eb-8035-079213d3606e.png)

I think it should be bigger below than off to the side or remove it. WDYT?